conseil-common/src/main/scala/tech/cryptonomic/conseil/common/mantis/domain/Bytecode.scala

```diff
@@ -0,0 +1,95 @@
+package tech.cryptonomic.conseil.common.mantis.domain
```

Comment:
A nice way of remove repetition for `mantis.domain` and  `ethereum.domain` is to move those to: `./conseil-common/src/main/scala/tech/cryptonomic/conseil/common/ethereumlike/domain`